LDIFGenerator/build.gradle

```
@@ -2,11 +2,17 @@ plugins {
     id 'application'
 }
 
-mainClassName = "com.willeke.LDIFGenerator"
+mainClassName = "com.willeke.ldap.ldif.LDIFGenerator"
 executableDir = ""
 
+repositories {
+    mavenCentral()
+}
+
 dependencies {
-    compile deps.log4j
+  compile group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.1'
+  compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.1'
+  compile group: 'org.apache.logging.log4j', name: 'log4j-1.2-api', version: '2.1'
```

`compile` configurations should be replaced with `implementation to support Gradle 7+ (Cf. https://stackoverflow.com/a/66910991/11931787)

```suggestion
  implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.1'
  implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.1'
  implementation group: 'org.apache.logging.log4j', name: 'log4j-1.2-api', version: '2.1'
```